client/src/js/services/account.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+/**
+ * @description 
+ *
+ * @returns 
+ */
+
+angular.module('bhima.services')
+  .factory('Account', Account);
+  
+Account.$inject = ['$http'];
+
+function Account($http) {  
+  
+  function list() { 
+    var path = '/accounts?list=\'full\'';
+
+    return $http.get(path)
+      .then(extractData);
+  }
+
+  function create (account){
+    var path = '/accounts';
+    return $http.post(path, account).then(extractData);
+  }
+
+  function getAccountDetails (list, id) { 
+    return list.filter(function (line){return line.id === id;})[0];
+  }  
+
+  return {
+    create : create,
+    getAccountDetails : getAccountDetails,
+    list : list  
+  };
+}
+
+
+
+// Utility method - pass only data object to controller
+// TODO Use shared utility service
```

Comment:
There is already a [service function for this operation](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/services/util.js#L9).  It is `util.unwrapHttpResponse`.  Could you use it in each service that does this?